backend/api/tokenVerificationApi.js

```diff
@@ -1,9 +1,9 @@
-const config = require('../config')
```

Comment:
Mix of commonjs / es6 imports means I need to implement this workaround in js files for config